src/components/Admin/ListUser/ListUser.js

```diff
@@ -21,6 +22,8 @@ const cookies = new Cookies();
 
 const TabPane = Tabs.TabPane;
 
+const Search = Input.Search;
```

Comment:
`const Search = { Input }`

Reply:
@anshumanv antdesign follows the above syntax and changing it to `const Search = { Input }` will throw errors on the console

Reply:
Okay, I see. Looks good then. :+1:

Reply:
Can you please post the error? I'm interested in knowing what it throws. @PrP-11

Reply:
Thanks @PrP-11  :sparkle: